array_family/flex_ext.py

```diff
@@ -1172,6 +1176,11 @@ def map_centroids_to_reciprocal_space(self, experiments, calculated=False):
             else:
                 sel_expt = self["id"] == i
 
+            u_mat = matrix.sqr((1, 0, 0, 0, 1, 0, 0, 0, 1))
+
+            if expt.crystal and crystal_frame:
+                u_mat = matrix.sqr(expt.crystal.get_U())
+
```

Comment:
I think it would be clearer to have
```
overall_rotation = fixed_rotation
if expt.crystal and crystal_frame:
    overall_rotation *= matrix.sqr(expt.crystal.get_U())
```
then use `overall_rotation` below

Reply:
Refactored following this suggestion in 71137e5595aac1d7864b1eb8234e2b5077b70ff4